clusters/psi/Makefile

```
@@ -3,11 +3,19 @@ SHELL=/usr/bin/env bash -o errexit
 all:
 
 TMPDIR ?= /tmp
+CLUSTER ?= app.ci
+SA_NAMESPACE ?= ci
+API_SERVER_URL ?= "https:\\/\\/api.ci.l2s4.p1.openshiftapps.com:6443"
 
-serviceaccounts = release-schedules-cm-updater sync-rover-groups-updater
+serviceaccounts = sync-rover-groups-updater
 
 $(serviceaccounts):
-	oc --context app.ci sa create-kubeconfig -n ci $@ | sed 's/$@/app.ci/g' > "$(TMPDIR)/sa.$@.app.ci.config"
+ifeq ($(TOTATE), true)
```

Is this supposed to be:
```suggestion
ifeq ($(ROTATE), true)
```